stargz/fs.go

```diff
@@ -80,13 +79,21 @@ const (
 	opaqueXattr               = "trusted.overlay.opaque"
 	opaqueXattrValue          = "y"
 	stateDirName              = ".stargz-snapshotter"
-	defaultLRUCacheEntry      = 5000
-	defaultResolveResultEntry = 3000
+	defaultLRUCacheEntry      = 100
+	defaultResolveResultEntry = 100
```

Comment:
From my experience, LRU by entry has few problems:
1.it cannot limit the memory usage precisely;
2.it introduce a global lock when doing the least-recent-update checking
may consider some cache based on ring buffer to avoid global lock, since this comes to filesystem, cache matters. I personally recommend VictoriaMetrics/fastcache.

Reply:
Thanks for the feedback! I agree with these issues and think we can move on more effective cache implementation including fastcache. Because the migration should require huge modification, let's keep it out of scope on this PR and discuss it on another issue/PR thread?

Reply:
Sorry for late reply, I was tied on some other work yesterday. Read through the code, nice work, left some reivew.

Reply:
Benchmarking test against some popular images is continuously measured in our CI https://github.com/containerd/stargz-snapshotter/actions?query=workflow%3ABenchmark+branch%3Amaster . One of the results is shown on our [README](https://github.com/containerd/stargz-snapshotter/blob/master/README.md).
Do you have ideas for `some existing test suite` ?

Reply:
And thanks for the feedbacks!:smile:

Reply:
Not that familiar with `test suite`, I comment when I learn more about it.

Reply:
Thanks!